spec/features/admin_node_update_feature_spec.rb

```diff
@@ -41,20 +41,30 @@
     end
     # rubocop:enable RSpec/MultipleExpectations
 
+    # rubocop:disable RSpec/ExampleLength
     scenario "User clicks on 'Reboot to update'", js: true do
-      allow(::Velum::Salt).to receive(:call)
+      allow(::Velum::Salt).to receive(:call).and_return(true)
 
       # clicks on "Update admin node"
       find(".update-admin-btn").click
 
+      # wait modal to appear
+      wait_until { page.has_text?("Reboot to update") }
+
       # clicks on "Reboot to update"
       find(".reboot-update-btn").click
 
-      # rubocop:disable RSpec/MessageSpies
-      expect(::Velum::Salt).to receive(:call)
-      # rubocop:enable RSpec/MessageSpies
+      wait_until { page.has_text?("Rebooting...") }
```

Comment:
This is no different than the following line - If it's passing because of this line it means the max wait for the next line isn't long enough, so just increase it there -  `expect(page).to have_content("Rebooting...", wait: 10)`